src/compiler/clang.rs

```diff
@@ -184,6 +185,7 @@ counted_array!(pub static ARGS: [ArgInfo<gcc::ArgData>; _] = [
     take_arg!("-fprofile-use", PathBuf, Concatenated('='), ClangProfileUse),
     take_arg!("-fsanitize-blacklist", PathBuf, Concatenated('='), ExtraHashFile),
     take_arg!("-gcc-toolchain", OsString, Separated, PassThrough),
+    flag!("-gcodeview", PreprocessorArgumentFlag),
```

Comment:
-gcodeview is not a preprocessor flag

Reply:
Agree, should I add a PassThroughFlag here?